app.py

```diff
@@ -10,5 +10,53 @@ def prompt_user_input():
 
 input = prompt_user_input()
 
+entries = []
+
+def add_entry():
+    print "Enter the item"
+    entry = raw_input()
+    entries.append(entry)
+
+def delete_entry():
+    print "Enter the element to be deleted"
+    to_delete = int(raw_input()) -1
+    del entries[to_delete]
+
+def check_entries():
+    y = 0
+    for x in entries:
+        print "Whether %s is done?" %(x)
```

Comment:
What is this?

Here it asks whether each of the entries is done from the user and prints the output.

That's not what I had asked for. When checking entries, you should first show the entries list and ask the user which of the entries in the list needs to be marked as checked. The user enters an input(say 2).

Now onwards, whenever you print the entries, you need to show 2 as marked, like below:
```
1. Daily chores
2. Goto church - checked
3. Learn python
```